server/src/mindustry/server/ServerControl.java

```diff
@@ -659,6 +659,16 @@ private void registerCommands(){
                 err("That IP/ID is not banned!");
             }
         });
+        
+        handler.register("pardon", "<ID>", "Pardons a votekicked player by ID and allows them to join again.", arg -> {
+            PlayerInfo info = netServer.admins.getInfo(arg[0]);
```

Comment:
Use `getInfoOptional`; `getInfo` will always create an info if it does not exist, and may never return null.